src/components/StyleGuide/Forms.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import { getCategory } from "../../utils/componentCategories";
+import getCSSVariable from "../../helpers/getCSSVariable";
+import Card from "../Card/Card";
+import List from "../List/List";
+import { ListTypes } from "../List/ListTypes";
+
+<Meta
+  title={getCategory("Forms")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/?node-id=10734%3A2768",
```

Comment:
Related to the above, it's fine to remove this and leave the "Figma Reference" below.